src/components/Image/index.native.tsx

```diff
@@ -36,24 +29,16 @@ function Image({source, isAuthTokenRequired, session, ...rest}: ImagePropsWithOn
             {...rest}
             source={imageSource}
             onLoad={(evt) => {
-                if (typeof source === 'number' || !source.uri) {
-                    return;
-                }
-                const {width, height} = evt.nativeEvent;
-                dimensionsCache.set(source.uri, {width, height});
-                if (rest.onLoad) {
-                    rest.onLoad(evt);
-                }
+                rest.onLoad?.(evt);
             }}
         />
     );
 }
 
 Image.displayName = 'Image';
 Image.resizeMode = RESIZE_MODES;
```

Comment:
`Image.resizeMode = RESIZE_MODES;` is still here.

Reply:
Removed!